pkg/controller/tuned/tuned_controller.go

```diff
@@ -122,6 +179,20 @@ func (r *ReconcileTuned) syncServiceAccount(tuned *tunedv1.Tuned) error {
 	} else {
 		// Do not update service account as it leads to accumulating secrets; see BZ1714484
 		glog.V(2).Infof("Tuned ServiceAccount already exists, skipping update")
+
+		isUpgrade, err := r.isOperandUpgrade(tuned)
```

Comment:
The secret management is in the `syncServiceAccount()` function because it is the first in the reconcile after the tuned fetch?

Reply:
Yes.  The secret management is not relevant until you start syncing the daemonset, so anything before that should be good.